Sortable.js

```diff
@@ -466,12 +471,28 @@
 				_css(ghostEl, 'position', 'fixed');
 				_css(ghostEl, 'zIndex', '100000');
 
-				rootEl.appendChild(ghostEl);
+				document.body.appendChild(ghostEl);
```

Comment:
GhostEl should be into rootEl, otherwise we will get the wrong display using CSS (`.root-block .dragged-item { ... }`).